ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -35,7 +35,9 @@ const withAdapter = Component =>
         <Component
           state={this.state}
           options={this.props.options}
-          setState={(...args) => this.props.model.set(...args)}
+          setState={(...args) => {
+            this.props.model.set(...args)
```

Comment:
Any reason for this change?

Reply:
I added the extra brackets because I put a console.log in there at one point, and I just forgot to take the brackets out